tests/test_managelist.py

```diff
@@ -0,0 +1,45 @@
+"""Test the ManageListProtocol Handler in agent."""
+from aries_staticagent import StaticConnection
+import pytest
+
+
+@pytest.mark.asyncio
+async def test_retrievelist(connection: StaticConnection):
+    await connection.send_async(
+        {
+            "@type": "https://example.com/" "manage-list/0.1/add",
+            "item": "First test string.",
+        }
+    )
+    await connection.send_async(
+        {
+            "@type": "https://example.com/" "manage-list/0.1/add",
+            "item": "Second test string.",
+        }
+    )
+    await connection.send_async(
+        {
+            "@type": "https://example.com/" "manage-list/0.1/add",
+            "item": "Third test string.",
+        }
+    )
+    reply = await connection.send_and_await_reply_async(
+        {
+            "@type": "https://example.com/" "manage-list/0.1/get-list",
+        },
+        return_route="all",
+    )
+    assert reply["item"] == [
+        "First test string.",
+        "Second test string.",
+        "Third test string.",
+    ]
```

Comment:
Minor recommendation :slightly_smiling_face: 
```python
items = ["first", "second", "third"]
for item in items:
    await connection.send_async(
        {
            "@type": "https://example.com/manage-list/0.1/add",
            "item": item
        }
    )
```
Then the assertion would look like:
```python
assert reply["item"] == items
```